view/src/App.tsx

```diff
@@ -1,86 +1,148 @@
-  const [selectedDataset, setSelectedDataset] = useState<Dataset | undefined>(
-    undefined
-  );
-  const [loading, setLoading] = useState(true);
-
-  useEffect(() => {
-    axios
-      .get(
-        `${API_BASE_URL}/api/getDataset?id=${window.location.pathname.slice(1)}`
-      )
-      .then((response) => {
-        if (response.data) {
-          handleReceiveDataset(response.data);
-        }
-        setLoading(false);
-      })
-      .catch(console.error);
-  }, []);
-
-  useEffect(() => {
-    document.title =
-      page === "welcome" ? "Upload Dataset" : "Data Visualization";
-  }, [page]);
-
-  function handleReceiveDataset(newDataset: Dataset) {
-    setSelectedDataset(newDataset);
-    setDatasets([...datasets, newDataset]);
-    setPage("graph");
-    setLoading(false);
-  }
-
-  function handleNewDataset() {
-    setPage("welcome");
-    setSelectedDataset(undefined);
-  }
-
-  function handleSelectDataset(dataset: Dataset) {
-    setSelectedDataset(dataset);
-    setPage("graph");
-  }
-
-  return (
-    <div className="">
-      <a href="#main-content" className="skip-link">
-        Skip to main content
-      </a>
-      <Navbar
-        datasets={datasets}
-        selectedDataset={selectedDataset}
-        onSelectDataset={handleSelectDataset}
-        onNewDataset={handleNewDataset}
-      />
-
-      {/* Main Content Area */}
-      <main id="main-content" role="main">
-        {loading ? (
-          <div className="w-screen flex justify-center">
-            <span className="loading loading-spinner text-success w-16 h-16"></span>
-          </div>
-        ) : page === "welcome" ? (
-          <WelcomePage
-            onDataset={handleReceiveDataset}
-            onSubmit={() => setLoading(true)}
-          />
-        ) : page === "graph" && selectedDataset ? (
-          <GraphPage dataset={selectedDataset} />
-        ) : (
-          <></>
-        )}
-      </main>
-    </div>
-  );
+    const [page, setPage] = useState<"landing" | "welcome" | "graph">("landing");
+    const [datasets, setDatasets] = useState<Dataset[]>([]);
+    const [selectedDataset, setSelectedDataset] = useState<Dataset | undefined>(undefined);
+
+    useEffect(() => {
+        document.title =
+            page === "landing"
+                ? "Welcome to Bias Visualizer"
+                : page === "welcome"
+                    ? "Upload Dataset"
+                    : "Data Visualization";
+    }, [page]);
+
+    function handleNewDataset() {
+        setPage("welcome");
+        setSelectedDataset(undefined);
+    }
+
+    return (
+        <div className="min-h-screen">
+            <Navbar
+                datasets={datasets}
+                selectedDataset={selectedDataset}
+                onSelectDataset={(dataset) => setSelectedDataset(dataset)}
+                onNewDataset={handleNewDataset}
+                isLandingPage={page === "landing"}
+            />
+
+            {/* Main Content Area */}
```

Comment:
This seems to be a landing page, but there's another landing page in LandingPage.tsx